cloud/imports_test.go

```diff
@@ -0,0 +1,12 @@
+//go:build offline
```

Comment:
Yeah, a lot of go tooling doesn't understand build tags very well. `gopls` is also still ignorant of tags, which makes editors pretty bad at syntax highlighting of tagged files.

Does the test itself need to be behind a build tag? Personally, I _usually_ only put tests behind a build tag if running them would cause problems - for example, tests that take a long time or require a network.